front/app/components/Form/SingleSelectControl.tsx

```diff
@@ -40,7 +40,7 @@ const SingleSelectControl = ({
   return (
     <>
       <FormLabel
-        htmlFor={id}
+        htmlFor={sanitizeForClassname(id)}
```

Comment:
Yes, it's coming from Jsonforms Control props didn't check the docs on that one, I thought it'd be a sanitized version of the path, and it's not, it's simply a unique identifier for the control, it actually resembles the path a lot but with / instead of ., or maybe they changed the path format in the upgrade ? I'll check the docs and pick this one or the path, whathever makes more sense.